data/fields/crossing.json

```diff
@@ -1,5 +1,13 @@
 {
     "key": "crossing",
     "type": "combo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+          "traffic_signals": "Traffic signals and marking",
+          "uncontrolled": "Marking only",
+          "unmarked": "No Traffic Signals, no marking",
```

Comment:
These values are using a mix of Title Case and Sentence case. I agree that title case can be annoying for long values, but the fields should at least be internally consistent.